Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -665,7 +660,7 @@ def parseInput(String[] cliArgs){
 		}
 
 		if(opts.t == false) {
-			println("*! Error: tarFilename is only optional when no build report order is specified")
+			println("*! [ERROR] tarFilename is only optional when no build report order is specified")
 			System.exit(3)
```

Comment:
I think, this should used `rc`, right? And there are a couple of more `exits` in the validation .